File: lesson_5_1/app/src/main/java/com/kalashnyk/denys/simplemvpapp/data/database/entity/UserEntity.java
package com.kalashnyk.denys.simplemvpapp.data.database.entity;

import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
public class UserEntity extends RealmObject {
    @SerializedName("id")
    @PrimaryKey
    private Long mId;

    @SerializedName("name")
    private String mName;

    @SerializedName("surname")
    private String mSurname;

    @SerializedName("fathername")
    private String mFathername;

    public UserEntity() { }

    @Override
    public String toString() {
        return "UserEntity{" +
                "mId=" + mId +
                ", mName='" + mName + '\'' +
                ", mSurname='" + mSurname + '\'' +
                ", mFathername='" + mFathername + '\'' +
                '}';
    }

    public Long getId() {
        return mId;
    }

    public void setId(Long mId) {
        this.mId = mId;
    }

    public String getName() {
        return mName;
    }

    public void setName(String mName) {
        this.mName = mName;
    }

    public String getSurname() {
        return mSurname;
    }

    public void setSurname(String mSurname) {
        this.mSurname = mSurname;
    }

    public String getFathername() {
        return mFathername;
    }

    public void setFathername(String mFathername) {
        this.mFathername = mFathername;
    }
}

File: lesson_7/app/src/main/java/com/kalashnyk/denys/kotlinsample/repository/database/entity/UserEntity.kt
package com.kalashnyk.denys.kotlinsample.repository.database.entity

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "users")
data class UserEntity(
    @PrimaryKey(autoGenerate = true)
    @SerializedName("id")
    val id: Int,
    @SerializedName("name")
    val name: String,
    @SerializedName("surname")
    val surname: String,
    @SerializedName("fathername")
    val fathername: String
)


